app/models.py

```diff
@@ -312,5 +312,15 @@ class HTML(db.Model):
 	value = db.Column(db.Text)
 
 class ZipSearch(db.Model):
-	zip_code = db.Column(db.String(5), primary_key = True)
+	id = db.Column(db.Integer, primary_key = True)
```

Comment:
Hmm, why did you change this? I thought using the zip as the primary key made a lot of sense.